NoSoft-SA/nspack
# Hotfix/add packing end date to seasons

Summary of this change:
Changed `CalenderRepo.for_select_seasons` function to filter seasons on new column `packing_end_date`.



CHANGELOG entry:

### Added
- `seasons.packing_end_date` migration, CRUD and grid column.
### Changed
- The function `for_select_seasons` now returns seasons where `packing_end_date` is greater than the current date (instead of previously filtering on `end_date`.

**How to test this code via the UI:**
Create/edit any record which has a seasons dropdown, such as production runs, and see if it filter correctly on `packing_end_date`
`packing_end_date` should default to `end_date` if left blank in seasons masterfile.
